apps/web/src/utils/file.ts

```diff
@@ -19,3 +19,20 @@ export function readFile(
     }
   })
 }
+
+export function downloadFile(url: string, name: string) {
+  const downloadLink = document.createElement('a')
+
+  // some browsers don't support the a[download] solution
+  if (typeof downloadLink.download === 'undefined') {
+    // 'window.location.href = downloadLink' no long works due to some Chrome policies update
+    return
+  }
```

Comment:
I think that we could just always use the append and remove approach if that works on all browsers. That way we can remove this `if` statement here - I'm not fussy about using this new feature in particular.

Thanks for going after this though, I learned something new! I always just used the append approach.

Reply:
I tested and Briefer doesn't work on Internet Explorer. `if` statement removed.